tests-e2e/fixture/binding/fixture.go

```diff
@@ -105,6 +105,81 @@ func HaveStatusGitOpsDeployments(gitOpsDeployments []appstudiosharedv1.BindingSt
 	}, BeTrue())
 }
 
+func HaveGitOpsDeploymentsWithStatusProperties(gitOpsDeployments []appstudiosharedv1.BindingStatusGitOpsDeployment) matcher.GomegaMatcher {
+
+	// compare compares two slices, returning true if the fields other than the CommitID are equal, regardless of the order of elements in the slices
+	compare := func(a []appstudiosharedv1.BindingStatusGitOpsDeployment, b []appstudiosharedv1.BindingStatusGitOpsDeployment) string {
+		if len(a) != len(b) {
+			return "lengths don't match"
+		}
+
+		for _, aDeployment := range a { // Expected
+			match := false
+			for _, bDeployment := range b { // Actual
+				// Compare exact strings but only check the length of the Commit ID since it is not deterministic.
+				if aDeployment.ComponentName == bDeployment.ComponentName && aDeployment.GitOpsDeployment == bDeployment.GitOpsDeployment &&
+					aDeployment.GitOpsDeploymentHealthStatus == bDeployment.GitOpsDeploymentHealthStatus &&
+					aDeployment.GitOpsDeploymentSyncStatus == bDeployment.GitOpsDeploymentSyncStatus &&
+					len(bDeployment.GitOpsDeploymentCommitID) > 0 && len(bDeployment.GitOpsDeploymentCommitID) <= 40 {
```

Comment:
:+1:

Reply:
I hope the commit hash lengths don't increase in the future?  I noticed they are 40 characters in length.